src/appsettings/settings.py

```diff
@@ -1206,13 +1206,20 @@ def check(self):
         """
         super(NestedDictSetting, self).check()
         errors = []
-        for subsetting in self.settings.values():
-            try:
-                subsetting.check()
-            except ValidationError as error:
-                errors.extend(error.messages)
-        if errors:
-            raise ValidationError(errors)
+        try:
+            self.raw_value
```

Comment:
I'd store the `raw_value` to a variable, so it isn't evaluated twice.

Reply:
Fixed (commited as fixup, maybe squash tem when merging)